content/02.body.md

```diff
@@ -61,9 +61,36 @@ Further discussion of this approximate edge prior and an derivation are availabl
 
 ## Results
 
-![Discrimination figure](){#fig:discrimination width="100%"}
+### Edge prior
+
+In the first prediction task, we computed three features---the XSwap edge prior, an approximation to the edge prior, and the (scaled) product of source and target node degree---on networks from Hetionet.
+We used the features in attempting to reconstruct the 20 networks themselves.
+The XSwap-derived edge prior is able to reconstruct many of the networks with a high level of performance, as measured by the AUROC.
+Of the 20 individual networks we extracted from Hetionet, 17 had an edge prior self-reconstruction AUROC >= 0.95, with the highest reconstruction AUROC at 0.9971 (Compound–downregulates–Gene edge type).
+Meanwhile, the lowest self-reconstruction performance (AUROC = 0.7697, Disease–localizes–Anatomy edge type) occurred in the network having the fewest node pairs.
+
+![AUROC of network reconstruction by prediction task.
+  The edge prior shows strong performance for network reconstruction when computed on the original (Task 1) and sampled (Task 2) networks.
+  The performance reduction from computing features on sampled networks is real but far smaller compared to a new degree distribution.
+  ](https://github.com/zietzm/xswap-analysis/raw/862d9504ec8546ea998285a708ab53a6c4f2fe2f/img/auroc_dists.png){#fig:discrimination width="50%"}
+
+The three features used in the comparison are highly rank correlated (median > 0.99999 across metaedges).
+Because AUROC takes only rank into account, the three features have very similar AUROC reconstruction performance values for the first, second, and third prediction tasks (max difference < 0.027).
+Nonetheless, we found that the edge prior exhibited marginally superior discriminatory performance in 12 of 20 networks.
+While the discriminatory abilities are similar, the features are very different in their levels of calibration---the ability of the model to correctly estimate class membership probabilities.
+We find that the edge prior is very well calibration for all networks in the first and second tasks, and it shows the best calibration of the three features for each of the prediction tasks (Figure {@fig:calibration}A).
+As the edge prior is not based on the networks' true edges, these results indicate that degree sequence alone is highly informative and can allow the reconstruction of edges, and they validate the use of the edge prior for this purpose.
+
+![**A.** The edge prior shows superior calibration to the other two features.
+  Calibration curves for full network reconstruction.
+  20 networks shown are from Hetionet.
+  **B.** Calibration curves for sampled network reconstruction. The edge prior shows superior calibration, though the levels of calibration decrease for all three features.
+  20 networks shown are from Hetionet.
+  **C.** Individual Hetionet edge type calibration metrics.
+  Lower scores indicate better calibration.
```

Comment:
I don't understand what is shown on this plot. Is the Y axis the error in the calibration?

Reply:
Instead of using "calibration" as a noun, could you try writing this with saying one is calibrated better or worse than the others? It might be a little bit clearer to read.